pkg/build/api/types.go

```diff
@@ -19,6 +19,12 @@ const (
 	BuildCloneAnnotation = "openshift.io/build.clone-of"
 	// BuildPodNameAnnotation is an annotation whose value is the name of the pod running this build
 	BuildPodNameAnnotation = "openshift.io/build.pod-name"
+	// BuildJenkinsStatusJSONAnnotation is an annotation holding the Jenkins status information
+	BuildJenkinsStatusJSONAnnotation = "openshift.io/jenkins-status-json"
```

Comment:
@bparees @jim-minter i thought that we are adding them just now, didn't realizes that they are something sent from Jenkins.